SignalServiceKit/src/Account/TSAccountManager.m

```diff
@@ -290,11 +290,6 @@ - (void)verifyAccountWithCode:(NSString *)verificationCode
                                                                                forNumber:phoneNumber
                                                                             signalingKey:signalingKey
                                                                                  authKey:authToken];
-    void (^completedRegistrationBlock)() = ^{
-        [self didRegister];
-        [TSSocketManager requestSocketOpen];
```

Comment:
1. didRegister is now called only after PushTokens have been uploaded.
2. no need to requestSocketOpen since TSSocketManager observes the registrationDidChange notification.